### PR TITLE
Replace @Autowired and @Qualifier annotations 

### PR DESCRIPTION
**Pull request by ahoffer**
Replaced `@Autowired` with `@Inject`
Attempted to replace `@Bean` with `@Named`, Spring failed to provide most of the beans.  Rather than leave a mix of `@Bean` and `@Named` I used `@Bean` consistently.

### CODE REVIEW

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

File: ingest/build.gradle [hunk 1]
```diff
@@ -13,10 +13,11 @@ dependencies {
     implementation(Dependencies.springBootStarterWeb())
     implementation(Dependencies.commonsIO())
     implementation(Dependencies.ingestAPI())
-    implementation(Dependencies.transformAPI())
+    implementation(Dependencies.javaxInject())
     implementation(Dependencies.springActuator())
     implementation(Dependencies.swagger())
     implementation(Dependencies.swaggerUi())
+    implementation(Dependencies.transformAPI())
```
Comment (ahoffer) [inline ingest/build.gradle]: Alphabetized

File: ingest/src/main/java/com/connexta/ingest/config/StoreClientConfiguration.java [hunk 2]
```diff
@@ -28,7 +28,7 @@ public RestTemplate nonBufferingRestTemplate() {
 
   @Bean
   public StoreClient storeClient(
-      @NotNull @Qualifier("nonBufferingRestTemplate") RestTemplate restTemplate,
+      @NotNull @Named("nonBufferingRestTemplate") RestTemplate restTemplate,
```
Comment (ahoffer) [inline ingest/src/main/java/com/connexta/ingest/config/StoreClientConfiguration.java]: `@Named` can replace `@Qualifier`

File: multi-int-store/build.gradle [hunk 3]
```diff
@@ -18,6 +18,8 @@ configurations.all {
 
 dependencies {
     implementation(Dependencies.awsS3())
+    implementation(Dependencies.ingestAPI())
```
Comment (emmberk) [inline multi-int-store/build.gradle]: Why does the mis need the Ingest API?

Comment (ahoffer) [reply multi-int-store/build.gradle]: It doesn't. That is a copy and paste error.

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Pull request merged**